lib/OrdinaryDiffEqSSPRK/src/OrdinaryDiffEqSSPRK.jl

```diff
@@ -0,0 +1,25 @@
+module OrdinaryDiffEqSSPRK
+
+import OrdinaryDiffEq: alg_order, alg_adaptive_order, calculate_residuals!,
+                       beta2_default, beta1_default, gamma_default,
+                       initialize!, perform_step!, @unpack, unwrap_alg,
+                       calculate_residuals, ssp_coefficient,
+                       OrdinaryDiffEqAlgorithm, ispredictive,
+                       OrdinaryDiffEqMutableCache, OrdinaryDiffEqConstantCache,
+                       OrdinaryDiffEqAdaptiveAlgorithm, uses_uprev,
+                       default_controller, PIDController,
```

Comment:
```suggestion
```

Clean up what's unused here, seems like it's just a copy paste with extra things carried in.